packages/dx-chart-core/src/plugins/series/computeds.js

```diff
@@ -166,27 +154,29 @@ const getUniqueName = (list, name) => {
 
 // TODO: Memoization is much needed here.
 // Though "series" list never persists, single "series" item most often does.
-const createPoints = (argumentField, valueField, data) => {
+const createPoints = (argumentField, valueField, data, props) => {
   const points = [];
   data.forEach((dataItem, index) => {
     const argument = dataItem[argumentField];
     const value = dataItem[valueField];
     if (argument !== undefined && value !== undefined) {
-      points.push({ argument, value, index });
+      points.push({
+        argument, value, index, ...props,
+      });
     }
   });
   return points;
 };
 
-export const addSeries = (series, data, palette, props) => {
+export const addSeries = (series, data, palette, props, restProps) => {
   // It is used to generate unique series dependent attribute names for patterns.
   // *symbolName* cannot be used as it cannot be part of DOM attribute name.
   const index = series.length;
   return [...series, {
     ...props,
     name: getUniqueName(series, props.name),
     index,
-    points: createPoints(props.argumentField, props.valueField, data),
+    points: createPoints(props.argumentField, props.valueField, data, restProps),
     palette, // TODO: For Pie only. Find a better place for it.
     color: props.color || palette[index % palette.length],
```

Comment:
`color` should be merged to `restProps` - otherwise there will be no `color` property in a series point.
`restProps` should be enumerated here instead of `color`.

Actually there is `color` property in a series point. But it is so because of `{...restProps}` in *point-collection.jsx*.

Let's add note in *point-collection.jsx* saying that `restProps` are used because of `getAnimatedStyle` and `scales`.